content/docs/ui/account-and-settings/upgrading-your-account.md

```diff
@@ -15,43 +15,41 @@ Whether you’re a startup or a large enterprise, SendGrid has several plans tai
 
 ## Upgrade your SendGrid plan
 
-### Choose a plan
+Before you can fully use your upgraded account, you need to complete some verification steps for security purposes.
 
-*Email API*
+<call-out type="warning">
 
-1. Decide the plan you upgrade to by comparing your current contact storage usage and email sends to the available plans [here](https://sendgrid.com/pricing).
-1. Navigate to your [**Account Details**](https://app.sendgrid.com/account/products) page, and then click on the **Your Products** tab.
-1. Under **Email API**, select **Change Plan**.
-1. Find the plan you would like to upgrade to and then click **Select**.
-1. Add your payment information in the popup window and then click **Submit**.
+If you signed up for a Twilio SendGrid account prior to 9/17/2020, your upgrade steps and verification process will look different from what's described below. 
```

Comment:
I think you can update this to say '... 9/17/20, you can skip to Choose Your Plan' and link to that section lower on the page.